README.md

```diff
@@ -254,3 +254,7 @@ Note: The [Patchwork](https://github.com/FlorentRevest/vscode-patchwork) and
 [Git Send Email](https://github.com/FlorentRevest/vscode-git-send-email)
 extensions are homegrown and maintained in other git repositories.
 Pull-requests or bug reports are also welcome there.
+
+## LPC Talk
+[SLIDES](https://lpc.events/event/17/contributions/1614/attachments/1210/2474/VSCode%20for%20kernel%20development.pdf)
```

Comment:
I'd prefer if this were a paragraph like "This setup was presented at the Linux Plumbers 2023 conference. The slides can be found here and the video below:"